cmd/zoekt-indexserver/main.go

```diff
@@ -290,15 +348,19 @@ func main() {
 		}
 	}
 
-	_, err := readConfigURL(opts.mirrorConfigFile)
-	if err != nil {
-		log.Fatalf("readConfigURL(%s): %v", opts.mirrorConfigFile, err)
-	}
+	if *dynamicIndexServer {
```

Comment:
This was easier to implement as "either dynamic webserver" OR "config file with polling" since the 2 seemed like completely different approaches. I think this partly makes me inclined towards a separate binary but then we've gotta come up with a new name 🤷

Reply:
Yeah I think if they are that different a new binary is a legitimate approach (it is what we did with zoekt-sourcegraph-indexserver). What you are implementing sounds more like a `zoekt-indexserver` than what we had before. Maybe if you wanna split it up (happy with starting off with same binary), the name for the old functionality is `zoekt-indexd` to take some sort of unix inspiration?

Reply:
This was moved to a separate binary now. I was a little concerned about renaming the old library without causing confusion so we went with `zoekt-dynamic-indexserver`. I agree `zoekt-indexd` is a clearer name for the previous functionality. I'm happy to take your lead on what you think is a reasonable risk in terms of breaking changes but renaming binaries will at least cause some headache for people with automated building and deployment of these binaries so it could just be easier to avoid it.